Makefile

```diff
@@ -145,10 +145,19 @@ ifndef DEPLOY_ENV
 	$(error DEPLOY_ENV is undefined)
 endif
 
+dump-inventory:
```

Comment:
Isn't this the equivalent (mostly) to ?:
```
ansible/inventory-list.json:
 	cd ansible && ansible-inventory --list > inventory-list.json
```

Let's try to use the makefile features where possible, I know we're abusing it, but still

Reply:
Yes, it's the better way, i chose the other way because of shorter name, will change it